agents/plugins/apache_status

```diff
@@ -162,6 +176,16 @@ def main():
     if not servers:
         return 0
 
+    overwrite_conf = {'custom': config.get("CUSTOM_ADDRESS_OVERWRITE", {})}
+    if config.get('ENABLE_OMD_SITE_DETECTION') and os.path.exists('/usr/bin/omd'):
+        for line in os.popen('omd sites').readlines():
+            sitename = line.split()[0]
+            site_raw_conf = \
```

Comment:
with open(.....) as fh:
   site_raw_conf = fh.readlines()

Was passiert bei Exceptions?

Reply:
Habe ich auf das with umgebaut. Aber macht das wirklich einen unterschied in dieser Art von Script? Bezüglich Exceptions, sollte hier eine Exception passieren, stimmt was generell nicht auf dem System was zu beheben wäre. Oder soll ich einfach alles abfangen? Dann könnte einfach ein leeres Objekt zurück gegeben werden. Finde ich aber, macht die Fehlersuche auch schwerer.